Sanity/pkcs11/single-encrypted-disk/runtest.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/clevis/Sanity/pkcs11/single-encrypted-disk
+#   Description: tests the clevis pkcs#11 luks functionality using a single encrypted disk
+#   Author: Martin Litwora <mlitwora@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2024 Red Hat, Inc.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/bin/rhts-environment.sh || :
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+
+setup_luks() {
```

Comment:
No problem, I moved it.

Reply:
But I still se it in test in working commit :D

Reply:
Oh, yeah. I moved only the simple  `luks_setup()` to the utils.sh. I did not want to move this specific one, as it feels a little bit more specific to this test case and I do not think it could be used somewhere else. But if you want to move this one as well, then I move it as well.

Reply:
We have discussed this on Slack channel. I also renamed the test specific function to `setup_luks_encrypted_single_disk()`